### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by kavyakunder**
Wishlist feature:

-Add to wishlist from product card
-In wishlist section, remove from wishlist

https://user-images.githubusercontent.com/81958043/161892266-143643cc-ca2a-49ec-9b16-d761f13977a2.mp4

### CODE REVIEW

File: src/components/WishList/WishListCard.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+function WishListCard({ product }) {
+  const { _id, name, price, img } = product;
+
+  const { wishDispatch } = useWishlist();
+  return (
+    <>
+      <div className="wishlist-card" id={_id}>
+        <div className="container-img">
+          <img className="wishlist-img" src={img} alt="product" />
```
Comment (shivampandey0) [inline src/components/WishList/WishListCard.js]: `alt="product"` is more generic, use name as alt here `alt={name}`

**Review by shivampandey0 — APPROVED** [review]
Excellent use of semantic tags where possible. Clean code, easy to understand.
Good to merge

**Review by Tanay-J — APPROVED** [review]
Functionality mentioned in the PR description are working. Good work at keeping the code clean and structured. This has made the code very easy to review.
Suggestion: You may use utility classes for font size, this would make the text sizing across the app consistent

File: public/index.html [hunk 2]
```diff
@@ -24,6 +24,11 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+      href="https://cdn.jsdelivr.net/gh/hung1001/font-awesome-pro@4cac1a6/css/all.css"
+      rel="stylesheet"
+      type="text/css"
+    />
     <link rel="stylesheet" href="https://robocon-ui.netlify.app/styles.css" />
 
     <title>React App</title>
```
Comment (Hamdrive) [inline public/index.html]: Cleanup the boilerplate comments, and also update the title of the page

**Review by Hamdrive — COMMENTED** [review]
Great implementation of reducer concept to handle wishlist updates
Will suggest to do cleanup of boilerplate code

File: src/components/WishList/WishListCard.js (hunk 1 above)
Comment (kavyakunder) [reply src/components/WishList/WishListCard.js]: done!

File: public/index.html (hunk 2 above)
Comment (kavyakunder) [reply public/index.html]: done!

**Pull request merged**